src/main/resources/content/tf/AWS-Customer-Master-Key-(CMK)-rotation-is-not-enabled.json

```diff
@@ -1,6 +1,6 @@
 {
   "severity": "medium",
-  "rule": "$.resource[*].aws_kms_key[*].*[?(@.enable_key_rotation != 'true')] exists",
+  "rule": "$.resource[*].aws_kms_key exists and ( $.resource[*].aws_kms_key[*].*[*].enable_key_rotation anyFalse or  $.resource[*].aws_kms_key[*].*[*].enable_key_rotation anyNull)",
```

Comment:
LGTM. Checks for existence, then any False boolean values, then subsequently null results